base/types/gfsptask/gc.go

```diff
@@ -313,12 +313,15 @@ func (m *GfSpGCStaleVersionObjectTask) InitGCStaleVersionObjectTask(priority cor
 	redundancyIndex int32,
 	integrityChecksum []byte,
 	pieceChecksumList [][]byte,
-	version, timeout int64) {
+	version int64,
+	objectSize uint64,
```

Comment:
objectSize What does this field do in the gc task？

Reply:
we need to copy the integrate meta from the shadow one. ObjectSize is needed